aequilibrae/starts_logging.py

```diff
@@ -34,4 +35,14 @@ def StartsLogging():
     return logger
 
 
+def cleaning():
+    p = tempfile.gettempdir() + "/aequilibrae_*"
+    for f in glob.glob(p):
+        try:
+            os.unlink(f)
```

Comment:
you can also just use pathlib and let it do the error ignoring, have i mentioned how much I hate exceptions

Reply:
I also hate exceptions, but there is no way to check if a file is in use somewhere else in the system before attempting to remove it